pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
[Per this answer](https://stackoverflow.com/a/30234912), I ran `sudo apt-get install tesseract-ocr` on a Ubuntu VM that had the failing test cases and it fixed them. It now passes all test on both my Windows machine and my Ubuntu VM (provided `tesseract-ocr` is installed). I don't know that this can be done as a built in dependency in Java.

It seems that the error is not a `TesseractException`, but rather an `UnsatisfiedLinkError`. I can add a catch for that and then log a message telling the user to run the command. Would that be an acceptable solution?